sdk/avalon_sdk/ethereum/ethereum_listener.py

```diff
@@ -220,25 +229,53 @@ def __init__(self, config):
             callback(normalized_event, *kargs, **kwargs)
```

Comment:
Are we still using the callback with the updated event handling logic ?

Reply:
This callback is still in place for connector interaction or where multiple events are expected